File: Number Series/Basic/Addition.c
#include<stdio.h>

void main() {

	int num1,num2;
	printf("Enter number 1: ");
	scanf("%d",&num1);

	printf("Enter number 2: ");
	scanf("%d",&num2);
	
	//addtion
	print("Addition = %d\n",num1+num2);

	//sub
	if(num1 > num2)
		print("Substraction = %d\n",num1 - num2);
	else
		print("Substraction = %d\n",num2 - num1);
}

File: Number Series/Basic/Addition.java
import java.io.*;
import java.lang.System.*;
class Program {
	public static void main(String args[]) {

		BufferedReader br = new BuferedReader (new InputStreamReader(System.in));

		int num1 = Integer.parseInt(br.readLine());
		int num2 = Integer.parseInt(br.readdLine());
		
		out.println("Addition : "+ (num1+num2));

		if(num1 > num2)	
			out.println("Substraction: "+ (num1 - num2));
		else
			out.println("Substraction : "+ (num2 - num1));
		

	}
}


